.github/issue_template.md

```diff
@@ -1,12 +1,16 @@
-### IMPORTANT ###
-This is not a support forum, it's a bug tracker. If you don't understand the difference, please do NOT open an issue. For questions and support, review recent postings on the john-users mailing list at https://www.openwall.com/lists/john-users/ and then subscribe to the list at https://www.openwall.com/john/#lists and post your message to the list by e-mailing `john-users at lists.openwall.com`.
+### Checklist ###
+- [ ] :1st_place_medal: I've read and understood these instrucions;
+  - This is not a support forum, it's a bug tracker. For questions and support, review postings on the [john-users mailing](https://www.openwall.com/lists/john-users/).
```

Comment:
`s/mailing/mailing list/`, then I'm happy with this. Let's see what @solardiz thinks as well.

Reply:
I removed a couple of duplicate lines. We can put them back if needed.